### PR TITLE
Addition of parent span if exists.

### PR DESCRIPTION
**Pull request by shubhammishra13**
The buildSpan method creates spans for mongo operations , i had a use case where these operations should be part of the overall trace for one request.I have added the changes where we first check if there is an existing parent span present or not.If yes then the mongo specific spans add the currently active span as the parent span.

### CODE REVIEW

**Comment by malafeev** [comment]
@shubhammishra13 could you revert all formatting?
it's hard to find your changes except reformatting?
also please don't use "*" in imports.

**Comment by shubhammishra13** [comment]
@malafeev i have fixed the formatting , can you review the changes.

File: opentracing-mongo-common/src/main/java/io/opentracing/contrib/mongo/common/TracingCommandListener.java [hunk 1]
```diff
@@ -172,10 +172,14 @@ Span buildSpan(CommandStartedEvent event) {
         .buildSpan(mongoSpanNameProvider.generateName(event))
         .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_CLIENT);
 
-    Span span = spanBuilder.start();
-    for (SpanDecorator decorator : decorators) {
-      decorator.commandStarted(event, span);
-    }
+        if (tracer.activeSpan() != null) {
```
Comment (malafeev) [inline opentracing-mongo-common/src/main/java/io/opentracing/contrib/mongo/common/TracingCommandListener.java]: there is no need explicitly add active span as parent.
it is done automatically.
you can verify that if you run your tests without this change.

Comment (shubhammishra13) [reply opentracing-mongo-common/src/main/java/io/opentracing/contrib/mongo/common/TracingCommandListener.java]: Yes seems to work even without the change. Thanks!

**Pull request closed without merging**